### PR TITLE
core/services/ocr2/plugins/ocr2keeper/evmregister/v21/upkeepstate: use sqlutil instead of pg.QOpts

### PR DESCRIPTION
**Pull request by jmank88**
https://smartcontract-it.atlassian.net/browse/BCF-2978

### CODE REVIEW

File: core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/upkeepstate/store.go [hunk 1]
```diff
@@ -320,7 +320,9 @@ func (u *upkeepStateStore) cleanup(ctx context.Context) error {
 func (u *upkeepStateStore) cleanDB(ctx context.Context) error {
 	tm := time.Now().Add(-1 * u.retention)
 
-	return u.orm.DeleteExpired(tm, pg.WithParentCtx(ctx), pg.WithLongQueryTimeout())
+	ctx, cancel := context.WithTimeout(sqlutil.WithoutDefaultTimeout(ctx), time.Minute)
```
Comment (dimriou) [inline core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/upkeepstate/store.go:323]: Should we remove the timeout completely?

Comment (jmank88) [reply core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/upkeepstate/store.go:323]: How come? I think these are still necessary to replace the default timeout which may be too short.

Comment (dimriou) [reply core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/upkeepstate/store.go:323]: Just wondering if it's ok to never timeout, compared to what we have now. I don't have a strong preference, just noticed the difference.

Comment (jmank88) [reply core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/upkeepstate/store.go:323]: This change is preserving the original "long" timeout of `1m`.

**Pull request merged**